assets/src/dashboard/app/api/useTemplateApi.js

```diff
@@ -0,0 +1,136 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useState, useMemo, useCallback } from 'react';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import getAllTemplates from '../../templates';
+import { APP_ROUTES } from '../../constants';
+
+function reshapeTemplateObject(isLocal) {
+  return ({ id, title, tags, colors, createdBy, description, pages }) => ({
+    isLocal,
+    id,
+    title,
+    createdBy,
+    description,
+    status: 'template',
+    modified: moment('2020-04-07'),
```

Comment:
Should this be a new date?

Reply:
Good call, I'll update our json templates so the date can be pulled from the data instead of hardcoded here.